recipes/default.rb

```diff
@@ -107,7 +107,7 @@
         client = RestClient.get 'http://127.0.0.1:4000/directory'
       rescue
         sleep 10
-        puts "Still waiting for boulder to start.. #{times * 10} seconds"
+        print "\nStill waiting for boulder to start.. #{times * 10} seconds"
```

Comment:
I think, although minor, that the original was better because it is more concise. Why did this change?

Reply:
Printing on the console was broken, it was merged into the line before this block, and had an extra newline afterwards. This just shifts where the newline is printed.